#!/usr/bin/env python
#
# Copyright (c) 2024 Numurus, LLC <https://www.numurus.com>.
#
# This file is part of nepi-engine
# (see https://github.com/nepi-engine).
#
# License: 3-clause BSD, see https://opensource.org/licenses/BSD-3-Clause
#

APP_NAME = 'Nav_Pose_Publisher' # Use in display menus
DESCRIPTION = 'Application for publishing the current NEPI nav pose solution in different standard formats'
PKG_NAME = 'nepi_app_navpose_publisher'
APP_FILE = 'nav_pose_publisher.py'
NODE_NAME = 'app_nav_pose_publisher'
RUI_FILES = []
RUI_MAIN_FILE = ''
RUI_MAIN_CLASS = ''
RUI_MENU_NAME = ''


